### PR TITLE
Fix move field m2o binary

### PR DESCRIPTION
**Pull request by legalsylvain**
Hi, 
I try to fix a minor bug in move_field_m2o function of openupgrade framework that occures if we try to migrate binary_field with undefined data. (In my case, move of image field from product.product to product.template).

Kind regards.

### CODE REVIEW

File: openerp/openupgrade/openupgrade.py [hunk 1]
```diff
@@ -861,7 +865,12 @@ def default_func(cr, pool, id, vals):
                 field_old_model, table_old_model, m2o_field_old_model, ko_id))
         cr.execute(query)
         if binary_field:
-            vals = [str(x[0][:]) for x in cr.fetchall()]
```
Comment (pedrobaeza) [inline openerp/openupgrade/openupgrade.py:864]: Better to put:

```
vals = [str(x[0][:]) if x[0] else False for x in cr.fetchall()]
```

Comment (legalsylvain) [inline openerp/openupgrade/openupgrade.py:864]: Thanks. Done.

**Comment by pedrobaeza** [comment]
:+1:

**Comment by StefanRijnhart** [comment]
Thanks! :+1:

**Pull request merged**